docs/doxygen-user/multi-user/installActiveMQ.dox

```diff
@@ -24,20 +24,27 @@ If you need the JRE, install it with the default settings.
 
 \subsection install_activemq_install_mq ActiveMQ Installation
 
-1. Extract the contents of the ActiveMQ archive folder to a location of your choice, bearing in mind that the files should be in a location that the running process will have write permissions to the folder. A typical folder choice would be similar to <i>C:\\Program Files\\apache-activemq-5.13.3</i>. Typically, it will ask for administrator permission to move the folder. Allow it if required. 
+<ol>
+<li>Extract the contents of the ActiveMQ archive folder to a location of your choice, bearing in mind that the files should be in a location that the running process will have write permissions to the folder. A typical folder choice would be similar to <i>C:\\Program Files\\apache-activemq-5.13.3</i>. Typically, it will ask for administrator permission to move the folder. Allow it if required. 
```

Comment:
Perhaps:
"ActiveMQ archive folder" => "ActiveMQ archive"
"in a location that the running process will have write permissions to the folder" => "in a location where the running process has write permissions"
"Typically, it will ask for administrator permission" => "The system may ask for administrator permission"